src/text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala

```diff
@@ -126,11 +126,12 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
 
   def looksLikeAVariable(mentions: Seq[Mention], state: State): Seq[Mention] = {
 
-
+    val punctuation = List(",", ".", "(", ")", "[", "]", "/", "~", "%")
     //returns mentions that look like a variable
     def passesFilters(v: Mention, isArg: Boolean): Boolean = {
       // If the variable was found with a Gazetteer passed through the webservice, keep it
       if ((v matches OdinEngine.VARIABLE_GAZETTEER_LABEL) && isArg) return true
+      if (v.words.length == 1 && punctuation.contains(v.words.head)) return false
```

Comment:
I think you could do this with the `isLetter` method, something like:
```
if (v.words.length == 1 && !(v.words.head.count(_.isLetter) > 0)) return false
```

then you wouldn't need a punct list

Reply:
That's a great idea! Thanks, @BeckySharp! Didn't occur to me!